### DISCUSSION TITLE
"deployment_status" isn't triggered

### DISCUSSION THREAD

**Discussion by cecilemuller** [community/community#26178] [Actions]
<p>Here’s a test case:</p>
<p><a href="https://github.com/wildpeaks/issue-deployment-status-action/tree/master/.github/workflows" rel="nofollow noopener">https://github.com/wildpeaks/issue-deployment-status-action/tree/master/.github/workflows</a></p>
<p>Two actions:</p>
<p> - <strong>Deploy</strong> triggered by <em>deployment</em> (when a Deployment is created using Github API v3): <strong>this runs fine</strong></p>
<p> - <strong>Notify</strong> triggered by <em>deployment_status</em>: <strong>this doesn’t run</strong></p>
<p>Action <strong>Deploy</strong> sends a <em>success</em> status:</p>
<p> - the Actions logs show the status was received successfully</p>
<p> - the Environments tab shows the status change and says it’s deployed</p>
<p>Yet the <strong>Notify</strong> action doesn’t show up in the Actions tab at all.</p>
<p>However, I managed to isolate the cause to the token used to set the Status:</p>
<p> - when using <strong>github.token</strong> , the status gets changed, but <strong>Notify isn’t triggered</strong></p>
<p> - when using a token from a user with <em>write</em> permission on the repo (the same one used to create the deployment), the status gets changed and <strong>Notify is triggered</strong></p>
<p>Is this by design ?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/cecilemuller">@cecilemuller</a> ,</p>
<p>Yes, this is by design.</p>
<p>With github token, an action in a workflow run can’t trigger a new workflow run. Please refer to official <a href="https://help.github.com/en/actions/reference/events-that-trigger-workflows#about-workflow-events" rel="nofollow noopener">doc</a> for more details.</p>
<p>Creating personal access token as secrets instead is the workaround.</p>
<p>Thanks.</p>
